Mathlib/Data/Fin/Basic.lean

```diff
@@ -49,7 +49,7 @@ This file expands on the development in the core library.
 * `Fin.orderIsoSubtype` : coercion to `{ i // i < n }` as an `OrderIso`;
 * `Fin.valEmbedding` : coercion to natural numbers as an `Embedding`;
 * `Fin.valOrderEmbedding` : coercion to natural numbers as an `OrderEmbedding`;
-* `Fin.succEmbedding` : `Fin.succ` as an `OrderEmbedding`;
+* `Fin.succEmb` : `Fin.succ` as an `OrderEmbedding`;
```

Comment:
Feel free to update `scripts/style-exceptions.txt` to bypass the CI failure on this file. Since you plan to split the file in a follow up PR.

Reply:
Thanks. I'll look at doing so.

Reply:
(I don't quite understand the syntax here - also, some other file, Mathlib/RingTheory/DedekindDomain/Ideal.lean, appears to be failing style checks, but I am not sure I modified this one: what gives?)